src/state_history_pg.hpp

```diff
@@ -251,83 +289,96 @@ inline std::string empty_to_sql<abieos::bytes>(pqxx::connection&, bool bulk) {
 }
 
 template <>
-inline std::string bin_to_sql<abieos::input_buffer>(pqxx::connection&, bool, abieos::input_buffer& bin) {
-    throw abieos::error("bin_to_sql: input_buffer unsupported");
+inline std::string bin_to_sql<eosio::input_stream>(pqxx::connection&, bool, eosio::input_stream& /*bin*/) {
+    eosio::check(false, "bin_to_sql: input_buffer unsupported");
+    return std::string{};
 }
 
 template <>
-inline std::string native_to_sql<abieos::input_buffer>(pqxx::connection&, bool bulk, const void* p) {
-    auto&       obj = *reinterpret_cast<const abieos::input_buffer*>(p);
+inline std::string native_to_sql<eosio::input_stream>(pqxx::connection&, bool bulk, const void* p) {
+    auto&       obj = *reinterpret_cast<const eosio::input_stream*>(p);
     std::string result;
     abieos::hex(obj.pos, obj.end, back_inserter(result));
     return quote_bytea(bulk, result);
 }
 
 template <>
-inline std::string empty_to_sql<abieos::input_buffer>(pqxx::connection&, bool bulk) {
+inline std::string empty_to_sql<eosio::input_stream>(pqxx::connection&, bool bulk) {
     return quote_bytea(bulk, "");
 }
 
 inline abieos::time_point sql_to_time_point(std::string s) {
     if (s.empty())
         return {};
     std::replace(s.begin(), s.end(), ' ', 'T');
-    return abieos::string_to_time_point(s);
+    abieos::time_point tp;
+    auto ts = eosio::json_token_stream(s.data());
```

Comment:
`s` isn't JSON

Reply:
Properly use template specialization conversion function previously added for block timestamp.